src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
```

Comment:
You are creating an instance of 'ColorSupplier' class but you don't use it. Please, remove this line.